### DISCUSSION TITLE
[feat]: Does ScrollArea support horizontal scrolling wheel?

### DISCUSSION THREAD

**Discussion by nvlong198** [shadcn-ui/ui#6098] [Ideas]
Scrolling to the end of content currently only works vertically, not horizontally.
Could you consider adding support for horizontal scrolling as well?

**Comment by minesunny** [comment]
any updates?

**Comment by minesunny** [comment]
> Scrolling to the end of content currently only works vertically, not horizontally. Could you consider adding support for horizontal scrolling as well?

Hay, I found a solution with [this](https://github.com/radix-ui/primitives/issues/2888#issuecomment-2513006859),it worked. but has some errors, the error is "e.preventDefault();", I removed it, and I don't know that waht  would be happen;
<img width="1077" alt="image" src="https://github.com/user-attachments/assets/3f01f906-234d-404c-a5c1-15b98c54c185" />
<img width="853" alt="image" src="https://github.com/user-attachments/assets/3bd378d9-e2bd-41a6-9b6e-37673b2988d5" />
<img width="1954" alt="image" src="https://github.com/user-attachments/assets/876db75b-4155-4741-843e-ecb81e551f56" />
